pacoloco.go

```diff
@@ -246,7 +466,7 @@ func sendCachedFile(w http.ResponseWriter, req *http.Request, fileName string, f
 	if err != nil {
 		return err
 	}
-
+	log.Printf("serving cached file %v", filePath)
```

Comment:
Adding a message when serving a cached file, it was missing (maybe it was intentional?)